scripts/development/m3_stack/dbnode_config.yml

```diff
@@ -0,0 +1,212 @@
+coordinator:
```

Comment:
re: dbnode_config.yaml and coordinator config here - i'm a little concerned about the duplication of configs - it's one more thing to keep in sync. Anyway we can merge with others we already have?

Reply:
Already sync'd with @robskillington and this is the best we can do for now until we can get some kind of templating system, I already did some kind of weird stuff to avoid having 3 different m3db configs

Reply:
sigh